DrawerKitDemo/DrawerKitDemo/PresentedViewController.swift

```diff
@@ -32,6 +32,10 @@ extension PresentedViewController: DrawerPresentable {
 
 //    var heightOfCollapsedDrawer: CGFloat {
 //        return 100
+//    }
+
+//    var fullExpansionBehaviour: DrawerConfiguration.FullExpansionBehaviour? {
+//        return .leavesCustomGap(gap: view.bounds.size.height - heightOfPartiallyExpandedDrawer)
```

Comment:
```suggestion
//        return .leavesCustomGap(gap: 100)
```

This example needs to work well when uncommented, no matter what configuration settings demo app has.
Otherwise, end user will find this as a bug, and thinks this library is broken.

So, let's set small enough `gap` to work well even when `supportsPartialExpansion = true`.